### DISCUSSION TITLE
Use Table in input Pipe instead of csv pipe

### DISCUSSION THREAD

**Discussion by durgesh312** [zinggAI/zingg#930] [Q&A] [answered]
How can I my databricks table as a input instead of a csv file?

**Comment by sonalgoyal** [comment, marked as answer]
you can try using the delta format in the Zingg Pipe. For example:

```
inputPipe = Pipe(name="nc_voters", format="delta")
inputPipe.addProperty("path","dbfs:/demos/ER/delta/nc_voters")

args.setData(inputPipe)
```

**Reply by a-mases** [reply]
Hello - along similar lines, is the path we put in inputPipe.addProperty different if the table is stored using Unity Catalog instead of DBFS? For example, the delta table I have in Unity Catalog has a storage location starting with s3:// - I tried using this link in .addProperty and I received a "could not read data" error. I also tried using a "catalog.schema.table" format and received the same error.
